sos/policies/distros/redhat.py

```diff
@@ -240,34 +230,6 @@ def __init__(self, sysroot=None, init=None, probe_runtime=True,
                          remote_exec=remote_exec)
         self.register_presets(RHEL_PRESETS)
 
-    @classmethod
-    def check(cls, remote=''):
-        """Test to see if the running host is a RHEL installation.
-
-            Checks for the presence of the "Red Hat Enterprise Linux"
-            release string at the beginning of the NAME field in the
-            `/etc/os-release` file and returns ``True`` if it is
-            found, and ``False`` otherwise.
-
-            :returns: ``True`` if the host is running RHEL or ``False``
-                      otherwise.
-        """
-
-        if remote:
-            return cls.distro in remote
-
-        if not os.path.exists(OS_RELEASE):
-            return False
-
-        with open(OS_RELEASE, "r", encoding='utf-8') as f:
-            for line in f:
-                if line.startswith("NAME"):
-                    (_, value) = line.split("=")
-                    value = value.strip("\"'")
-                    if value.startswith(cls.distro):
-                        return True
```

Comment:
We are changing this to direct `value`s match, but that is imho correct. `NAME` should be `"Red Hat Enterprise Linux Server"`.